bin/main.rb

```diff
@@ -0,0 +1,29 @@
+module UserInteraction
+    def get_rol
+        puts "\nWELCOME TO THE TIC TAC TOE GAME"
+        puts "==========================="
+        puts "Which player do you prefer?"
+        puts "1 for X or 2 for O"
+        puts "==========================="
+        $rol = gets.chomp.to_i
```

Comment:
Sorry but the **global variables** is not allowed

Reply:
done